### PR TITLE
build(frontend): update frontend packages 2024-08-21

### LINKED ISSUE #1750: 🐛 [Bug] Cannot close Meeting drawer with gesture
## 🐛 Bug

Cannot close Meeting drawer with gesture
image

### PR DESCRIPTION
**Pull request by Bettelstab**
## 🍰 Pullrequest
Updates
- typescript
- vue-tsc
- vuetify
- @typescript-eslint packages
- vite-plugin-checker
and makes some necessary changes.

Fixes #1750

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "vue": "3.4.35",
     "vue-i18n": "9.13.0",
     "vue3-toastify": "^0.2.2",
-    "vuetify": "^3.5.17",
+    "vuetify": "^3.7.0",
```
[the remark this answers is not in the document]
Comment (Bettelstab) [reply frontend/package.json:82]: There was some problem indeed, with the navigation drawer, and I fixed it.

File: frontend/src/components/tablesDrawer/TablesDrawer.vue [hunk 2]
```diff
@@ -76,10 +75,6 @@ const filteredItems = computed(() => {
 const currentLocation = ref(props.location)
 const isChangingOrientation = ref(false)
```
[the remark this answers is not in the document]
Comment (Bettelstab) [reply frontend/src/components/tablesDrawer/TablesDrawer.vue:76]: Mmhh I don't want to remove it, I think we still need it, yes.

File: frontend/src/pages/Guard.spec.ts [hunk 3]
```diff
@@ -16,7 +16,10 @@ describe('global route guard', () => {
 
   describe('unauthenticated', () => {
     it('throws and redirects', () => {
-      expect(() => guard({ hasToken: false } as PageContextServer)).toThrow(/* what exactly? */)
+      /* We expect redirect('/signin') to be thrown, but it's an object,
+         which is not an acceptable argument for toThrow(). */
```
Comment (roschaefer) [inline frontend/src/pages/Guard.spec.ts:20]: Whyyyy..

**Comment by roschaefer** [comment]
> The `toThrow()` test could be improved for sure, but seems a bit out of scope for this PR.

Using `try/catch` inside a test is a terrible thing to do and leads to false negatives (bug present but test green). That's why I insisted :slightly_smiling_face:

**Review by roschaefer — APPROVED** [review]
Alright, my requested changes have been implemented! :clap:

**Pull request merged**